tests/lib/rules/object-curly-spacing.js

```diff
@@ -229,7 +229,7 @@ ruleTester.run("object-curly-spacing", rule, {
                     message: "A space is required after '{'",
                     type: "ImportDeclaration",
                     line: 1,
-                    column: 11
+                    column: 12
```

Comment:
Based on my top comment, this message tells me that I need a space after letter `b` which is incorrect.
Correct: I need a space after `{`